tests/testthat/test-survival-fit-resamples.R

```diff
@@ -0,0 +1,375 @@
+suppressPackageStartupMessages(library(tidymodels))
+suppressPackageStartupMessages(library(censored))
+suppressPackageStartupMessages(library(baguette))
+
+skip_if_not_installed("parsnip", minimum_version = "1.1.0.9003")
+skip_if_not_installed("censored", minimum_version = "0.2.0.9000")
+skip_if_not_installed("tune", minimum_version = "1.1.1.9001")
+skip_if_not_installed("yardstick", minimum_version = "1.2.0.9001")
+
+test_that("resampling survival models with static metric", {
+  skip_if_not_installed("prodlim")
+
+  # standard setup start -------------------------------------------------------
+
+  set.seed(1)
+  sim_dat <- prodlim::SimSurv(500) %>%
+    mutate(event_time = Surv(time, event)) %>%
+    select(event_time, X1, X2)
+
+  set.seed(2)
+  split <- initial_split(sim_dat)
+  sim_tr <- training(split)
+  sim_te <- testing(split)
+  sim_rs <- vfold_cv(sim_tr)
+
+  time_points <- c(10, 1, 5, 15)
+
+  mod_spec <-
+    bag_tree() %>%
+    set_mode("censored regression")
+
+  rsctrl <- control_resamples(save_pred = TRUE)
+
+  # resampling models with static metrics --------------------------------------
+
+  stc_mtrc  <- metric_set(concordance_survival)
+
+  set.seed(2193)
+  rs_static_res <-
+    mod_spec %>%
+    fit_resamples(
+      event_time ~ X1 + X2,
+      resamples = sim_rs,
+      metrics = stc_mtrc,
+      control = rsctrl
+    )
```

Comment:
I think we should test what happens if you pass in `time_points` to `fit_resamples()` when they are not needed